notebook/API.livemd

```diff
@@ -11,8 +11,105 @@ base_url = IO.gets("Base URL") |> String.trim()
 ## Users
 
 ### GET /v1/me
+
 Used to fetch the current user's profile details
 
 ```elixir
 LivemanApi.get(base_url, "/v1/me")
 ```
+
+## Surveys
+
+### GET /v1/surveys
+
+Used to fetch all the available surveys.
+
+Returns a 200 OK response
+
+```elixir
+LivemanApi.get(base_url, "/v1/surveys")
+```
+
+Supports pagination by passing the page number and page size as params
+
+```elixir
+LivemanApi.get(base_url, "/v1/surveys", %{page: 2, page_size: 8})
+```
+
+### POST /v1/surveys
+
+Used to create an answer to a particular survey
+
+Requires a valid access_token in the header
+
+Returns a 201 Created response
+
+#### 201 OK
+
+```elixir
+headers = [access_token: "77564df8-7823-45b2-b9ef-38dac927ddfd"]
+
+body = %{
+  survey_id: "d5de6a8f8f5f1cfe51bc",
```

Comment:
Should we make this as a separate input above, maybe for survey_id, question_id? - but my concern is we can't do for the answer id array, right? 🤔

Reply:
That's right, unless we do some trimming and splitting.. Maybe we could have a separate code block dedicated for the body? 🤔